app/services/trainees/create_timeline_events.rb

```diff
@@ -58,38 +59,49 @@ def initialize(audit:)
     end
 
     def call
-      if create_single_event?
+      if action == "create"
```

Comment:
Should we put "create" and "destroy" in constants as you have done with `STATE_CHANGE`?